README.md

```diff
@@ -102,6 +105,18 @@ Select "Use an existing project"
 yarn dev:startlocaldb
 ```
 
+### 3.c Starting the dev server and seeding the emulator
+
+```sh
+yarn dev
+```
+
+If the emulator is missing the seed data try to run the following to make sure the correct project id is set:
+
+```sh
+yarn firebase --project="find-a-doc-japan" emulators:start
```

Comment:
is this command correct? I'm not sure if yarn will run this if it's not a script. Please double check. If it works, this is fine 👍

Reply:
actually, as I'm thinking about it, can we just add the project option to the `yarn dev:startlocaldb` command itself?

Reply:
@ermish I think it's a good idea to add it directly to the script!

Reply:
@ermish @theyokohamalife I double checked and the command is working. But I feel I should make it more clear when to run that command? 
Since you have to run `yarn firebase --project="find-a-doc-japan" emulators:start ` instead of `yarn dev:startlocaldb`  followed by `yarn dev`

Reply:
@Anissa3005 the idea is you can add the `--project="find-a-doc-japan"` to the `dev:startlocaldb` so you won't need to run your firebase command anymore 😄  then we can just run `dev:startlocaldb` then `yarn dev`

Reply:
@ermish I see since `dev:startlocaldb` is the firebase command of course. I will change this in the README

Reply:
@Anissa3005 What you'll need to do is update the command in package.json so it's `dev:startlocaldb --project="find-a-doc-japan"` and then remove the reference to it in the README.

Reply:
@theyokohamalife I have just added that, and it's working fine!